app/assets/scripts/views/analytics-admin-area.js

```diff
@@ -1,9 +1,13 @@
 'use strict';
 import React from 'react';
-import { t } from '../utils/i18n';
+import { connect } from 'react-redux';
+import { _ } from 'lodash';
```

Comment:
Isn't it `import _ from 'lodash';` instead? Not sure what issues this nested import would cause.